src/routes/protected/index.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
I suggest use only one component to ProtectedRoute and Public route there are so similar